### PR TITLE
adding drone-kubernetes-apply

### PR DESCRIPTION
**Pull request by naorlivne**
Created (yet) another kubernetes deployment plugin, the main thing is that I wanted one plugin that supports all type of kubernetes resources & allows declaring multiple resources in a single YAML with being able to dynamically modify variables in the YAML & couldn't find one so created my own.

### CODE REVIEW

**Comment by naorlivne** [comment]
Hi guys, 

Is anyone free to review this?

**Comment by naorlivne** [comment]
@tboerger Can I get a review of this?

**Comment by jones2026** [comment]
@naorlivne does this plugin support just using the kube config file?

File: content/naorlivne/drone-kubernetes-apply/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+date: 2019-08-27T00:00:00+00:00
+title: Kubernetes
```
Comment (tboerger) [inline content/naorlivne/drone-kubernetes-apply/index.md:3]: Title would fit better: Kubernetes Apply

File: content/naorlivne/drone-kubernetes-apply/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+author: naorlivne
+tags: [ deploy, kubernetes, docker ]
+logo: kubernetes.svg
+repo: naorlivne/drone-kubernetes-apply
+image: naorlivne/drone-kubernetes-apply
+---
+
+This plugin can be used to deploy applications to a Kubernetes cluster, it will create\update the given kubernetes deployments/cron jobs/pods/etc as needed.
```
Comment (tboerger) [inline content/naorlivne/drone-kubernetes-apply/index.md:11]: ...create or update the given Kubernetes manifest as needed.

File: content/naorlivne/drone-kubernetes-apply/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+Main difference between this plugins & other kubernetes Drone plugins is that it uses kubectl under the hood to allow multiple resources to be declared in a single YAML file & supports all resources types provided by Kubernetes.
```
Comment (tboerger) [inline content/naorlivne/drone-kubernetes-apply/index.md:13]: Plugin in singular, Kubernetes with an uppercase K, and please wie and instead of &

Comment (naorlivne) [reply content/naorlivne/drone-kubernetes-apply/index.md]: Fixed

File: content/naorlivne/drone-kubernetes-apply/index.md (hunk 2 above)
Comment (naorlivne) [reply content/naorlivne/drone-kubernetes-apply/index.md]: Changed

File: content/naorlivne/drone-kubernetes-apply/index.md (hunk 1 above)
Comment (naorlivne) [reply content/naorlivne/drone-kubernetes-apply/index.md]: Changed

**Comment by naorlivne** [comment]
> @naorlivne does this plugin support just using the kube config file?

Short answer is yes, long(er) answer is your the 2nd guy to ask for it in a week & it was a simple fix so I added support for it in the latest version that was released today.

@tboerger made the requested changes, please rereview/approve.

**Comment by naorlivne** [comment]
@tboerger / @bradrydzewski  can I get a review of this please? I see similar plugins referencing this ticket approved but as I prefer my approach better to my use-case I would really like seeing it made public

**Comment by naorlivne** [comment]
Hi all, 

This has been pending review for a year without a comment, is there something wrong or just falling through the cracks?

**Comment by naorlivne** [comment]
Hi @d1wilko, as you can see this has been open since 2020 waiting for a review from your team for over 2 years, while I appreciate consistency as the next dev can you guarantee to me that if I do the work of migrating this PR to the new format it will be reviewed in a reasonable timeframe? I'm not interested in wasting my time on this just for it to sit for another 2 years while your team ignores multiple review requests until the next format change.

**Pull request closed without merging**